go/lambda/MakeFunction/MakeFunction.go

```diff
@@ -0,0 +1,99 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// snippet-start:[lambda.go.make_function]
+package main
+
+// snippet-start:[lambda.go.make_function.imports]
+import (
+    "flag"
+    "fmt"
+    "io/ioutil"
+
+    "github.com/aws/aws-sdk-go/aws"
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/lambda"
+    "github.com/aws/aws-sdk-go/service/lambda/lambdaiface"
+)
+// snippet-end:[lambda.go.make_function.imports]
+
+// MakeFunction creates a Lambda function
+// Inputs:
+//     svc is a Lambda service client
+//     zipFile is the name of the zip file
+//     bucket is the name of the bucket
+//     function is the name of the Lambda function
+//     handler is the name of the function that is called in the Lambda function
+//     resource is the
```

Comment:
resource is the... what?

Reply:
Fixed (Amazon Resource Name (ARN) of the function's execution role)